.ci/validation.sh

```diff
@@ -90,7 +90,7 @@ no-error-pmd)
   echo "CS_version: ${CS_POM_VERSION}"
   mkdir -p .ci-temp/
   cd .ci-temp/
-  git clone https://github.com/pmd/pmd.git
+  git clone https://github.com/Abhishek-kumar09/pmd.git
```

Comment:
https://github.com/pmd/pmd/pull/2925 is merged, please restore original repo.

Reply:
Done